src/ui/components/SecondaryToolbox/SecondaryToolbox.css

```diff
@@ -55,11 +55,14 @@
   color: var(--tab-standard-color);
 }
 
+.secondary-toolbox-header button:hover .label {
+  color: var(--tab-standard-hover-color);
+}
```

Comment:
switches from a white bg on hover to a darker color which is less visually jarring